docs/src/handles.md

```diff
@@ -126,6 +136,20 @@ class_names = element_handle.get_attribute('class')
 assert 'highlighted' in class_names
 ```
 
+```csharp
+// Get the element handle
+var jsHandle = await page.WaitForSelectorAsync("#box");
+var elementHandle = jsHandle as ElementHandle;
```

Comment:
wondering about this. If we keep the `AsElementHandle` function, why not using that?
I like more the example version tho.